Bookish/Bookish.DataAccess/LibraryService.cs

```diff
@@ -32,29 +32,25 @@ public Book GetCopies(string isbn)
                 "LEFT JOIN Users ON Users.userId=Loans.userId " +
                 $"WHERE LibraryBooks.isbn='{isbn}';";
 
-            var bookDictionary = new Dictionary<string, Book>();
-
+            var outBook = new Book();
 
             var bookWithCopies = databaseConnection.Query<Book, BookCopy, Book>(
                     query,
                     (book, bookCopy) =>
                     {
-                        Book bookEntry;
-
-                        if (!bookDictionary.TryGetValue(book.isbn, out bookEntry))
+                        if (book.isbn != outBook.isbn)
```

Comment:
You can just check if `outBook == null` here I believe. And just initialise `outBook` to `null` when you declare it 🙂